packaging/scripts/preinstall.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+#
+# Executed before the installation of the new package
+#
+#   $1=install              : On installation
+#   $1=upgrade              : On upgrade
+
+set -e
+
+EVCC_USER=evcc
+EVCC_GROUP=evcc
+EVCC_HOME="/var/lib/$EVCC_USER"
+RESTART_FLAG_FILE=/var/lib/evcc/.restartOnUpgrade
+
+copyDbToUserDir() {
+  if [ -d /root/.evcc ] && [ ! -f /root/.evcc/.copiedToEvccUser ]; then
+    touch /var/lib/openhab2/.copiedToEvccUser
```

Comment:
wofür brauchen wir das?

Reply:
also v.a. unter dem Key openhab? Wäre es nicht besser, die Datei zu verschieben?

Reply:
Das openhab war natürlich ein copy/paste Fehler, sorry. Verschieben ist immer so eine Sache. Falls was schief geht, ist die Datei weg und wir brauchen weitere logik in post-install abort.. So bleibt das Original erhalten und ein revert zur vorherigen Version macht keine Probleme.

Das Flag dient dazu, zu markieren, dass wir die DB bereits kopiert haben. Damit sparen wir uns Unterscheidungen, welches genau die letzte Version war. Wenn es eine DB unter Root gibt UND die nicht als schon kopiert markiert ist, kopier sie rüber. Ist sie schon kopiert worden, kann sie einfach ignoriert werden.